resolver/GlobalPass.cc

```diff
@@ -61,7 +61,7 @@ bool resolveTypeMember(core::GlobalState &gs, core::SymbolRef parent, core::Symb
         typeMember.data(gs)->resultType = core::make_type<core::LambdaParam>(typeMember, untyped, untyped);
         return false;
     }
-    if (!my.isTypeMember() && !my.isTypeArgument()) {
+    if (!my.isTypeMember()) {
```

Comment:
I don't know why both of these were checked before. The implementation of these two are conflated elsewhere, so my assumption was that it was incorrect cargo culting.